### ISSUE TITLE
Make the app translatable

### ISSUE THREAD

**Issue by maicol07** [zend10/AL-chan#69]
I would like to create an Italian translation for AL Chan but it seems the app doesn't have support for multlanguage... I hope it will be added soon...

**Comment by zend10** [comment]
Yes, v.2.0 should be more supportive towards multi-language if I don't mess up.

**Comment by FateXBlood** [comment]
@zend10 
Please make the application to be translated in other languages. I want to translate AL-Chan in Nepali and Hindi. 

I recommend Weblate for this, as it will be very convenient to translate. (https://hosted.weblate.org/)

Most of the apps I use already support Nepali and Hindi. As such, it would be better if AL-Chan allows to do the same. It is alright if all the information in AniList is in English, we can simultaneously work on translating the app in our native language for a better experience.
